exporters/otlp/otlplog/otlploggrpc/client_test.go

```diff
@@ -450,6 +451,23 @@ func (c *grpcCollector) Collect() *storage {
 	return c.storage
 }
 
+// Addr returns the net.Addr c is listening at.
+func (c *grpcCollector) Addr() net.Addr {
```

Comment:
This is a private struct used exclusively within this package. We don't need this method, we can do the same as other tests do: directly use `c.listener.Addr().String()`.